src/hooks/protections/receive_net_message.cpp

```diff
@@ -235,7 +235,21 @@ namespace big
 			switch (msgType)
 			{
 			case rage::eNetMessage::MsgScriptMigrateHost: return true;
-			case rage::eNetMessage::MsgRadioStationSyncRequest: return true;
+			case rage::eNetMessage::MsgRadioStationSyncRequest:
+			{
+				static rate_limiter unk_player_radio_requests{2s, 2};
+
+				if (unk_player_radio_requests.process())
+				{
+					if (unk_player_radio_requests.exceeded_last_process())
+					{
+						// Make a translation for this new OOM kick protection
+						g_notification_service.push_error("PROTECTIONS"_T.data(), "OOM_KICK"_T.data());
```

Comment:
...So are you gonna make a translation for this?

Reply:
Don't forget to call session::add_infraction(player, Infraction::TRIED_KICK_PLAYER).
Also: There might be no need for a different notification text, imo.

Reply:
The current translation includes {} for a player name, and this pr handles the case where player is nullptr(not host) so we can't get the name of the player

Reply:
> Don't forget to call session::add_infraction(player, Infraction::TRIED_KICK_PLAYER). Also: There might be no need for a different notification text, imo.

To who? We don't have a player pointer.